api/server/server.go

```diff
@@ -111,6 +112,10 @@ type Config struct {
 
 // NewServer starts and returns a new server with the given configuration.
 func NewServer(conf Config) (*Server, error) {
+	if conf.FFSUseMasterAddr && !(len(conf.LotusMasterAddr) > 0 || conf.AutocreateMasterAddr) {
+		return nil, fmt.Errorf("FFSUseMasterAddr requires LotusMasterAddr or AutocreateMasterAddr to be provided")
+	}
+
```

Comment:
Here's the check to make sure we have a master address or will create on if `FFSUseMasterAddr` is `true`

Reply:
👍